pyrgg/engines/pyrgg.py

```diff
@@ -265,23 +247,18 @@ def logger(file, file_name, elapsed_time, input_dict):
     :return: None
     """
     try:
-        file.write(LOGGER_TEMPLATE.format(datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
-                                          file_name,
-                                          str(input_dict["vertices"]),
-                                          str(input_dict["edge_number"]),
-                                          str(input_dict["max_edge"]),
-                                          str(input_dict["min_edge"]),
-                                          str(bool(input_dict["direct"])),
-                                          str(bool(input_dict["sign"])),
-                                          str(bool(input_dict["multigraph"])),
-                                          str(bool(input_dict["self_loop"])),
-                                          str(is_weighted(input_dict["max_weight"],
-                                                          input_dict["min_weight"],
-                                                          bool(input_dict["sign"]))),
-                                          str(input_dict["max_weight"]),
-                                          str(input_dict["min_weight"]),
-                                          input_dict["engine"],
-                                          ENGINE_MENU[input_dict["engine"]],
-                                          elapsed_time))
+        text = f"Vertices : {str(input_dict['vertices'])}\n"
```

Comment:
I suggest using `.format()` instead of `f{}`

Reply:
The latest Python supporting version is `3.6` for pyrgg I guess we can use f-formats now.